package.json

```diff
@@ -7,40 +7,40 @@
     "url": "https://github.com/mozilla/chronicle/issues"
   },
   "dependencies": {
-    "bell": "2.0.0",
+    "bell": "2.1.0",
     "boom": "2.6.1",
     "commander": "2.6.0",
-    "convict": "0.6.0",
-    "elasticsearch": "3.1.1",
-    "embedly": "1.0.2",
-    "hapi": "8.1.0",
+    "convict": "0.6.1",
+    "elasticsearch": "3.1.3",
```

Comment:
elasticsearch changelog 3.1.1 -> 3.1.3: https://github.com/elasticsearch/elasticsearch-js/commits/v3.1.3

- lot of test-related changes and jenkins-related changes
- rev bluebird
- add support for 504 gateway timeout error handling